core/src/retransmit_stage.rs

```diff
@@ -314,19 +314,22 @@ fn retransmit(
             .fetch_add(retransmit_time.as_us(), Ordering::Relaxed);
         num_nodes
     };
-    fn merge<K, V>(mut acc: HashMap<K, V>, other: HashMap<K, V>) -> HashMap<K, V>
```

Comment:
I personally think this original form is more readable.
The function body is also shorter.

Reply:
One downside of the original form is that you have to specify all the generic parameters and traits, while for closure, the compiler can infer it.